core/test/EncryptedSender.js

```diff
@@ -1,12 +1,12 @@
 const { didContractThrow } = require("../../common/SolidityTestUtils.js");
-const { getRandomSignedInt, getRandomUnsignedInt } = require("../utils/Random.js");
+const { getRandomSignedInt, getRandomUnsignedInt } = require("../../common/Random.js");
```

Comment:
Where do you see `Random.js` being needed? (I'm totally fine with moving it either way - just curious)

Reply:
Please ignore - just remembered that we'd probably use it for salt generation.